Nautilus/Handlers/PDAHandler.cs

```diff
@@ -220,8 +220,7 @@ public static void AddEncyclopediaEntry(PDAEncyclopedia.EntryData entry)
         }
 
         PDAEncyclopediaPatcher.CustomEntryData[entry.key] = entry;
-        
-        if (uGUI.isMainLevel)
+        if(PDAEncyclopedia.initialized)
```

Comment:
Alright, this should really be in a different PR but looks like it should have been done a long time ago.

Reply:
But if you want it could be in a different PR so we can track such a major change more easily.

Reply:
And honestly there's no "if you want" to it, really should be organized into a separate PR.

Reply:
Hmmm, this PR doesn't work without this change.

Reply:
So they'd have to be merged at the same time.

Reply:
Done.